buildchow/templates/core-jobs.yaml

```diff
@@ -5,9 +5,11 @@
       master
     devstudio-stream:
       master
-    target-platform-version: 4.50.0.Beta1-SNAPSHOT
-    target-platform-version-maximum: 4.50.0.Beta1-SNAPSHOT
-    disabled: true
+    target-platform-version: 4.50.0.CR1-SNAPSHOT
+    target-platform-version-maximum: 4.50.0.CR1-SNAPSHOT
+    blockBuildWhenDownstreamBuilding: false
```

Comment:
It's already in the jobs, so yes. Also, since we removed inter-job linkages 2+ years ago, there's no way for Jenkins to know that base blocks openshift.